### PR TITLE
Update homepage to https for 23 Casks

### PR DESCRIPTION
**Pull request by miccal**
Update homepage to https for 23 Casks from the list [here](http://caskroom.victorpopkov.com/homebrew-cask/homepages.csv).

### CODE REVIEW

File: Casks/beyerdynamicvs.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
   url 'http://www.beyerdynamic.com/fileadmin/editors/downloads/virtualstudio/beyerdynamic-virtual-studio_04-13.zip'
   name 'beyerdynamic Virtual Studio'
-  homepage 'https://www.beyerdynamic.com/virtual-studio.html'
+  homepage 'https://europe.beyerdynamic.com/virtual-studio.html'
```
Comment (fanquake) [inline Casks/beyerdynamicvs.rb]: This one seems to redirect depending on region, so I think the original is correct. i.e following `https://www.beyerdynamic.com/virtual-studio.html` I end up at `http://asia-pacific-india.beyerdynamic.com/virtual-studio.html`

**Comment by miccal** [comment]
Some of the changes in this PR are now not required because of [this PR](https://github.com/caskroom/homebrew-cask/pull/26981) - what would be the best way to remove the Casks that have already been changed in #26981?

**Comment by victorpopkov** [comment]
Oops, didn't notice your PR. You just need to rebase. Need any help?

**Comment by miccal** [comment]
All fixed.

**Comment by victorpopkov** [comment]
Try running:
```bash
git pull upstream master
git checkout addhttps
git rebase master
git rebase --skip # on each conflict
```

**Comment by miccal** [comment]
Thanks @victorpopkov, but this PR had become too messy! No problem though, I will re-submit later after I look at your updated list.

**Pull request closed without merging**